extensions/amp-web-push/0.1/iframe.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { setStyle } from '../../../src/style';
+
+export default class IFrame {
```

Comment:
This is used as a helper abstraction when [creating the IFrame](https://github.com/OneSignal/amp-web-push/blob/63f8b5e096e688b4a7aab2c134c00b26977d2a4b/extensions/amp-web-push/0.1/web-push-service.js#L111) and when [loading the IFrame](https://github.com/OneSignal/amp-web-push/blob/63f8b5e096e688b4a7aab2c134c00b26977d2a4b/extensions/amp-web-push/0.1/web-push-service.js#L119).